src/core/public/core_app/status/components/metric_tiles.tsx

```diff
@@ -46,3 +53,7 @@ const formatMetric = ({ value, type }: Metric) => {
   const metrics = Array.isArray(value) ? value : [value];
   return metrics.map((metric) => formatNumber(metric, type)).join(', ');
 };
+
+const formatMetricId = ({ name }: Metric) => {
+  return name.toLowerCase().replace(' ', '');
```

Comment:
```suggestion
  return name.toLowerCase().replaceAll(' ', '');
```

`.replace()` only replaces the first space, which probably isn't what is intended.

Reply:
Ah yea, always forget the good old `/g`, thanks.